docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+version: '3'
+services:
+  cow-cms:
+    container_name: cow-cms
+    build: .
+    image: cow-cms:latest
+    restart: unless-stopped
+    env_file: .env
```

Comment:
cool! but shouldn't you explain how to create this file in the README? 

Pls, provide some basic instructions on how to run it for development and prod

Reply:
In principle, just like any docker compose, this project can be run by simply running `docker compose up`. However it is a great point, I'll add it 👍

Reply:
and not only in the redme, you have to add the test instructions in your PRs

Reply:
I still didn't see the README. Remember, to add instructions such as the commands to launch in dev, and prod. How you create the env (setup locally), and how to access the different services. The PR feels undocumented

Reply:
README is added.